bench/in.sicrackfoam_M1

```diff
@@ -0,0 +1,62 @@
+log		log.sicrackfoam  # Name of log file to write to
+
+#Test relaxation script for Nathan 
+
+boundary    p p p
+
+units       metal
+atom_style  atomic
+neighbor    2.0 bin
+neigh_modify    delay 5
+
+## create geometry
+
+read_data CrackBlock
+
+# define groups
+
+region	    mobile block -70.0 70.0 -70.0 70.0 INF INF
+group		mobile region mobile
+group		boundary subtract all mobile
+#set		    group boundary type 2
+
+pair_style eam
+pair_coeff * * /Users/bnathan2/Desktop/LAMMPS_git_Aug14_2019/potentials/Au_u3.eam

```

Comment:
please don't put absolute paths in input examples, and definitely not paths that won't work. also the data file is WAAAY too large for inclusion in the distribution and it should not be added into the bench folder. please also put a log command into example inputs.